README.md

```diff
@@ -91,6 +91,7 @@ These are the possible role variables - you only need to have a small set define
 
     symfony_project_branch: git branch, commit hash or version tag to deploy - defaults to master
     symfony_project_php_path: php
+    symfony_project_php_options: ''
```

Comment:
Should be "" instead of '' to be consistent :-)
Maybe add a short description about what the variable does - or add an example line on how it is intended to be used